recipes/vulkan-validationlayers/all/dependencies/dependencies-1.2.154.0.yml

```diff
@@ -0,0 +1,2 @@
+spirv-tools: "2020.5"
```

Comment:
yes, seems to be more convenient over having a mapping inside the recipe

Reply:
Yes, it looks a great improvement!